ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -21,6 +21,8 @@ class DemoListViewController: UITableViewController {
         let demoListViewController = DemoListViewController(nibName: nil, bundle: nil)
 
         let navigationController = UINavigationController(rootViewController: demoListViewController)
+        navigationController.navigationBar.isTranslucent = true
+
```

Comment:
@mischreiber I just noticed that setting the isTranslucent property to true affects the edges of the SwiftUI demo controllers:

![Screen Shot 2021-10-06 at 12 35 38 PM](https://user-images.githubusercontent.com/68076145/136271314-48aeb3b8-18bb-4b13-a293-6fb5f2be0c98.png)


iOS **13**
![Screen Shot 2021-10-06 at 12 21 05 PM](https://user-images.githubusercontent.com/68076145/136269151-876cd285-2574-4125-ade6-d198040ded77.png)



**iOS 15**
![Screen Shot 2021-10-06 at 12 20 15 PM](https://user-images.githubusercontent.com/68076145/136269023-1e4d77da-75ef-4e99-b4fa-e018ad07b93d.png)